webTest/index.html

```diff
@@ -0,0 +1,84 @@
+<!Doctype html>
+<html>
+
+<head>
+    <title> Test </title>
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<!-- background="background.png" -->
+
+<body>
+    <header>
+        <h1>CYF</h1>
+
+        <nav>
+            <ul>
+                <li> <a>Menu1</a> </li>
+                <li> <a>Menu2</a> </li>
+                <li> <a>Menu3</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="firstContainer">
+
+        <h1>About CYF </h1>
+        <h1>What We Do</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer tincidunt sollicitudin ultrices. Fusce dictum sem et varius porta. Aenean suscipit mauris sit amet pretium laoreet. Aliquam egestas est non nisi pulvinar, ut facilisis justo placerat. Fusce
+            fermentum lectus eget molestie tempor. Nam lobortis erat eget varius semper. Nullam molestie viverra nulla, eget accumsan augue maximus auctor. Maecenas placerat ante pharetra tellus cursus, fringilla sollicitudin lacus mollis.
+        </p>
+
+        <h3>Our Focus</h3>
+        <ul>
+            <li> Interactive </li>
+            <li> Mobile </li>
+            <li> Technology </li>
+        </ul>
+
+        <h3><a href="#">Click here To Download a PDF of our recent work</a></h3>
+        <h1>News</h1>
+    </div>
+
+    <div class="secandContainer">
+        <h2> <a id="showHide" href="#">Show/Hide news item </a></h2>
+        <div class="showOne">
+
+
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer tincidunt sollicitudin ultrices. Fusce dictum sem et varius porta. Aenean suscipit mauris sit amet pretium laoreet. Aliquam egestas est non nisi pulvinar, ut facilisis justo placerat. Fusce
+                fermentum lectus eget molestie tempor. Nam lobortis erat eget varius semper. Nullam molestie viverra nulla, eget accumsan augue maximus auctor. Maecenas placerat ante pharetra tellus cursus, fringilla sollicitudin lacus mollis.
+            </p>
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer tincidunt sollicitudin ultrices.
+            </p>
+        </div>
+
+    </div>
+
+    <form>
+        <h2>More Information</h2>
+        <h4>For more information about our product, please contact us using the form below</h4>
+        <h4> iteam marked* are required fields </h4>
+
+        <div>
+            <h3>Contact name*:</h3>
+            <input id="contName" type="textarea" />
+
+
+            <h3>Contact email address*:</h3>
```

Comment:
no need to abbreviate the IDs .. just make them **contactName**, it's better for readability if you're sharing code with other people or in a team.